### PR TITLE
Added upgrade menu lang

### PR DESCRIPTION
**Pull request by Superalex0102**

### CODE REVIEW

File: src/main/java/com/bgsoftware/wildstacker/menu/SpawnerUpgradeMenu.java [hunk 1]
```diff
@@ -126,7 +126,7 @@ protected Inventory buildInventory() {
                 ItemStack rawCurrentIcon = spawnerUpgrade.getIcon();
                 ItemMeta rawCurrentItemMeta = rawCurrentIcon.getItemMeta();
                 ItemStack currentIcon = new ItemBuilder(rawCurrentIcon)
-                        .withName("&6Current Upgrade: &7" + rawCurrentItemMeta.getDisplayName())
+                        .withName(Locale.UPGRADE_CURRENT + rawCurrentItemMeta.getDisplayName())
```
Comment (OmerBenGera) [inline src/main/java/com/bgsoftware/wildstacker/menu/SpawnerUpgradeMenu.java:129]: `Locale.UPGRADE_CURRENT`'s string is not the message itself.
Use Locale.UPGRADE_CURRENT#getMessage instead.

File: src/main/java/com/bgsoftware/wildstacker/menu/SpawnerUpgradeMenu.java [hunk 2]
```diff
@@ -143,7 +143,7 @@ protected Inventory buildInventory() {
 
                 ItemStack rawNextIcon = nextUpgrade.getIcon();
                 ItemStack nextIcon = new ItemBuilder(rawNextIcon)
-                        .withName("&6Next Upgrade: &7" + rawNextIcon.getItemMeta().getDisplayName())
+                        .withName(Locale.UPGRADE_NEXT + rawNextIcon.getItemMeta().getDisplayName())
```
Comment (OmerBenGera) [inline src/main/java/com/bgsoftware/wildstacker/menu/SpawnerUpgradeMenu.java:146]: `Locale.UPGRADE_NEXT`'s string is not the message itself.
Use Locale.UPGRADE_NEXT#getMessage instead.

**Review by OmerBenGera — CHANGES_REQUESTED** [review]
You haven't tested this before making the PR, were you?
Make sure you test it before you submit a PR for the next time.

File: src/main/resources/lang.yml [hunk 3]
```diff
@@ -114,4 +114,8 @@ ENTITY_INFO_FOOTER: '&8&m---------------&6&l WildStacker &8&m---------------'
 # Called when simulating stacking of two objects
 OBJECT_SIMULATE_CHOOSE_SECOND: '&6&lWildStacker &7Please click another object.'
 OBJECT_SIMULATE_SUCCESS_RESULT: '&6&lWildStacker &7Result: &aSUCCESS&7. They can stack together.'
-OBJECT_SIMULATE_FAIL_RESULT: '&6&lWildStacker &7Result: &c{0}&7. They cannot stack together.'
\ No newline at end of file
+OBJECT_SIMULATE_FAIL_RESULT: '&6&lWildStacker &7Result: &c{0}&7. They cannot stack together.'
+
+# Called on upgrade menu
```
Comment (OmerBenGera) [inline src/main/resources/lang.yml:119]: "Used in the upgrades menu" or something like that may be better for describing these messages.

**Review by OmerBenGera — CHANGES_REQUESTED** [review]

File: src/main/java/com/bgsoftware/wildstacker/menu/SpawnerUpgradeMenu.java (hunk 1 above)
Comment (Superalex0102) [reply src/main/java/com/bgsoftware/wildstacker/menu/SpawnerUpgradeMenu.java:129]: Sorry about that

File: src/main/resources/lang.yml (hunk 3 above)
Comment (Superalex0102) [reply src/main/resources/lang.yml:119]: I wanted to use the same format, but okay

**Comment by OmerBenGera** [comment]
I will test it on weekend and merge :)

**Comment by OmerBenGera** [comment]
It's not possible to run the WildStacker that is compiled by your repository.
I don't like the fact that it's not possible to run it, and that you haven't even tested it.
I will merge it for this time, however in the future I will reject pull requests like this.

Thanks for helping tho, appreciated :)

**Comment by Superalex0102** [comment]
I did test it and had no issues this time, I even changed the lang in the file

**Pull request merged**